### DISCUSSION TITLE
cannot translate specific strings in custom translations

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3479] [Q&A]
Hello Pretix team!
I have translated several strings to greek on behalf of a customer in a premises hosted environment.
I cannot translate the following 2 strings that are displayed in tht cart checkout procedure
"The items in your cart are reserved for you for %(minutes)s minutes." and in singular too.
"The items in your cart are no longer reserved for you. You can still complete your order as long as they’re available."

Both of the strings are placed in the same position of the cart. Check the screenshot below.
https://snipboard.io/4Zdj6a.jpg

**Comment by wiffbi** [comment]
It seems to me the strings in ui/cart.js:50 are not moved into the translation process. They are marked as fuzzy in my local installation. It is 

`msgid "The items in your cart are reserved for you for one minute."`
`msgid_plural "The items in your cart are reserved for you for {num} minutes."`

**Reply by raphaelm** [reply]
Yeah, the Greek translation is marked fuzzy:
https://github.com/pretix/pretix/blob/1f80e9ef8240c79da5a4087b1f3faee17ded548a/src/pretix/locale/el/LC_MESSAGES/djangojs.po#L683

But generally, it works fine:
https://github.com/pretix/pretix/blob/1f80e9ef8240c79da5a4087b1f3faee17ded548a/src/pretix/locale/de/LC_MESSAGES/djangojs.po#L646

The translation can be updated here:
https://translate.pretix.eu/translate/pretix/pretix-js/el/?q=state%3A%3Ctranslated+minute&sort_by=-priority%2Cposition&offset=6

**Comment by alemao8** [comment]
I would like to translate greek in my environment, so the djangojs.po file in my server looks like this

#: pretix/static/pretixpresale/js/ui/cart.js:50
msgid "The items in your cart are reserved for you for one minute."
msgid_plural "The items in your cart are reserved for you for {num} minutes."
msgstr[0] "Τα είδη στο καλάθι θα παραμείνουν δεσμευμένα για ένα λεπτό."
msgstr[1] "Τα είδη στο καλάθι θα παραμείνουν δεσμευμένα για {num} λεπτά."

The path of the djangojs.po is /var/pretix/venv/lib/python3.9/site-packages/pretix/locale/el/LC_MESSAGES/djangojs.po

Unfortunately, whatever change I try, nothing changes in the greek text. It remains untranslated.

**Reply by raphaelm** [reply]
Have you recompiled the translations and restarted the server?

**Comment by alemao8** [comment]
yes I have recompiled and restarted the pretix-web.service as in other cases with succesful translations.
Do u mean something else by _restarted the server_ ?

**Reply by raphaelm** [reply]
Ah you might also need the "rebuild" step

**Reply by raphaelm** [reply]
Like after an update

**Comment by alemao8** [comment]
The rebuild step was the key point!
(venv)$ python -m pretix rebuild
#systemctl restart pretix-web pretix-worker

Solved!
